doc-content/enterprise-only/cases/case-management-starting-itorders-in-showcase-proc.adoc

```diff
@@ -1,52 +1,45 @@
-[id='case-management-starting-itorders-in-showcase-proc-{context}']
-= Starting an IT Orders case in the Showcase application 
+[id='case-management-showcase-tasks-proc']
+= Starting dynamic tasks and processes
 
-You can start a new case instance for the IT Orders sample case management project in the Showcase application. 
+You can add dynamic tasks and processes to a case during run time. Dynamic tasks are a way to address changing situations, where an unanticipated change during the case requires a new task or process to be incorporated into the case.
 
-The IT Orders sample case management project includes the following roles:
+.Prerequisites
+* The {KIE_SERVER} is deployed and connected to {CENTRAL}.
+* The IT Orders project is deployed to {KIE_SERVER}.
+* The Showcase application `.war` file has been deployed within {CENTRAL}.
 
-* `owner` - the employee who is making the hardware order request. There can be only one of these roles.
-* `manager` - the employee's manager; the person who will approve or deny the requested hardware. There is only one manager in the IT Orders project.
-* `supplier` - the available suppliers of IT hardware in the system. There is usually more than one supplier.
-
-These roles are configured at the case definition level:
-
-.ITOrders Case Roles
-image::cases/case_roles.png[Case Roles]
-
-Assign users or groups to these roles when starting a new case file instance.
-
-.Prerequisites 
+.Procedure
+. Select *Order for IT hardware* case instance from the list to open it.
+. Under *Overview* -> *Actions* -> *Available*, click image:cases/dotdotdotbutton.png[] next to *New user task* or *New process task* to add a new task or process task.
++
+.Showcase dynamic actions
+image::cases/showcase-dynamic-actions.png[]
+. Click start to create a *New user task* and input the following values:
```

Comment:
Enter/input - I get it. You like input :-)

Reply:
I'm so confused